plugins/cisco_meraki_logs.yaml

```diff
@@ -0,0 +1,116 @@
+version: 0.0.1
+title: Cisco Meraki
+description: Log parser for Cisco Meraki
+parameters:
+  - name: listen_port
+    type: int
+    default: 5140
+  - name: listen_address
```

Comment:
seems like this should be listen_ip to match the other plugins

Reply:
It could be and IP or a Host name. I just has to be routable. Might be worth looking at previous plugins.